Makefile

```diff
@@ -212,6 +212,12 @@ test_sani: LDFLAGS = -fsanitize=address,undefined
 test_sani: LDLIBS = -lpthread -lgtest -lgmock -lgtest_main
 test_sani: $(TEST_SANI)
 
+# This target uses the file valid_config.conf as argument to run the program.
+.PHONY: valid_config
+valid_config: $(NAME)
```

Comment:
I woud just call it "run"
This one confilcts with target valgr. So you have to type vali and TAB.
I think make run is also easier to remember

Reply:
I agree, done